src/qibolab/native.py

```diff
@@ -348,8 +346,18 @@ class TwoQubitNatives:
     """Container with the native two-qubit gates acting on a specific pair of
     qubits."""
 
-    CZ: Optional[NativeSequence] = None
-    iSWAP: Optional[NativeSequence] = None
+    CZ: Optional[NativeSequence] = field(default=None, metadata={"symmetric": True})
+    CNOT: Optional[NativeSequence] = field(default=None, metadata={"symmetric": False})
+    iSWAP: Optional[NativeSequence] = field(default=None, metadata={"symmetric": True})
+
+    @property
+    def symmetric(self):
+        """Check if the defined two-qubit gates are symmetric between target
+        and control qubits."""
+        for fld in fields(self):
+            if not fld.metadata["symmetric"] and getattr(self, fld.name) is not None:
+                return False
+        return True
```

Comment:
This maybe would be more explicit as something like:
```suggestion
        return all(not fld.metadata["symmetric"] and getattr(self, fld.name) is not None for fld in fields(self))
```
(properly formatted, of course)

Just because in this way, you declare your intent from the beginning: check if all elements of a sequence satisfy a certain condition or not.